.env_SAMPLE

```diff
@@ -108,9 +122,9 @@ export WORDPRESS=http://www.consumerfinance.gov
 # TESTING (optional) - for handling testing of the site.
 ########################################################
 
-#export HTTP_HOST=localhost
-#export HTTP_PORT=8000
-#export SELENIUM_URL=http://localhost:4444/wd/hub
+#export SITE_DESC=cfgov
+#export TEST_HTTP_HOST=localhost
+#export TEST_HTTP_PORT=8000
```

Comment:
This originally came from needing an easy way to swap port values to/from the default, so I set it up as three step:
 1. unset the value
 2. set override (commented out by default)
 3. set default.

So the unset step (1) and the default step (3) would always be enabled and should remain untouched for consistency and ease of copying from `.env_SAMPLE`. To set the port, for example, L:127 would be uncommented and changed to `export TEST_HTTP_PORT=8001` and `source ./.env` would be run. To reset it back, comment out L:127 and re-run `source ./.env`. 

If that's too complicated (or doesn't work outside of bash) I'm happy to updated to whatever works for you'll. It's more important I think that the `process.env` calls are consolidated in one spot.